src/app/shared/address-input/address-input.component.ts

```diff
@@ -43,12 +43,13 @@ import { AddressService } from '../services/address/address.service';
     }
   ]
 })
-export class AddressInputComponent implements OnInit, AfterViewInit, OnDestroy, ControlValueAccessor, Validator {
+export class AddressInputComponent implements OnInit, AfterViewInit, OnDestroy, OnChanges, ControlValueAccessor, Validator {
   @Input() edit: boolean;
   @Input() address: Address;
   @Input() addFromProfile: boolean;
   @Input() isShowCommentInput = true;
   @Input() isFromAdminPage: boolean;
+  @Input() uneditableStatus: boolean;
```

Comment:
Agree with @Lokankara. I would use `isUneditableStatus` or `canEditStatus` names. The current name is a bit confusing, personally, for me.